README.md

```diff
@@ -1,288 +1,85 @@
-  </ul>
-</details>
-
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
-
-<!-- Features -->
-
-### Key Features <a name="key-features"></a>
-
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
-
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- GETTING STARTED -->
-
-## 💻 Getting Started <a name="getting-started"></a>
-
-> Describe how a new developer could make use of your project.
-
-To get a local copy up and running, follow these steps.
+To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
-In order to run this project you need:
+-Computer
 
-<!--
-Example command:
+-Internet
 
-```sh
- gem install rails
-```
- -->
+-Web Browser
 
 ### Setup
 
-Clone this repository to your desired folder:
-
-<!--
-Example commands:
-
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
+- Clone the repository to your computer, or you download the zip file containing the repository files.
 
-### Install
-
-Install this project with:
-
-<!--
-Example command:
-
-```sh
-  cd my-project
-  gem install
-```
---->
+- In order to use is possible to :
 
-### Usage
-
-To run the project, execute the following command:
+- Open editor terminal
 
-<!--
-Example command:
+- Digit "git clone https://github.com/zied2112/Awesome-books.git"
 
-```sh
-  rails server
-```
---->
+- Then "cd My BookStore"
 
-### Run tests
+- File editing can then be applied using any code editor.
 
-To run tests, run the following command:
+### Install
 
-<!--
-Example command:
+- `npm install`
 
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
+### Usage
 
 ### Deployment
 
-You can deploy this project using:
+- To view the page right from the files cloned to your computer, open the index.html file from any browser.
 
-<!--
-Example:
-
-```sh
-
-```
- -->
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- AUTHORS -->
-
-## 👥 Authors <a name="authors"></a>
-
-> Mention all of the collaborators of this project.
+## Authors
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@github](https://github.com/zied2112)
+- Twitter: [@twitter](https://twitter.com/AmorZied1996)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/zied-ben-amor-924908149/)
```

Comment:
- Please make sure your README is similar to that of the [microverse README Template](https://github.com/microverseinc/readme-template) and all sections are filled 👍🏽